core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -78,19 +79,20 @@ public String toString() {
 
     public ProjectConfig(String accountId, String projectId, String version, String revision, List<Group> groups,
                          List<Experiment> experiments, List<Attribute> attributes, List<EventType> eventType,
-                         List<Audience> audiences) {
-        this(accountId, projectId, version, revision, groups, experiments, attributes, eventType, audiences,
+                         List<Audience> audiences, boolean anonymizeIP) {
```

Comment:
we only need to add `anonymizeIP` to one `ProjectConfig` constructor. You can remove it here and call the other constructor with `false` as the default value.